flink-table/flink-table-planner/src/main/java/org/apache/flink/table/planner/plan/rules/logical/FlinkSemiAntiJoinFilterTransposeRule.java

```diff
@@ -79,7 +81,7 @@ public void onMatch(RelOptRuleCall call) {
                         join.getJoinType());
 
         final RelFactories.FilterFactory factory = RelFactories.DEFAULT_FILTER_FACTORY;
-        RelNode newFilter = factory.createFilter(newJoin, filter.getCondition());
+        RelNode newFilter = factory.createFilter(newJoin, filter.getCondition(), ImmutableSet.of());
```

Comment:
Deprecated in https://issues.apache.org/jira/browse/CALCITE-3183
in favor of a method with correlating vars arg